content/index.html

```diff
@@ -153,8 +158,13 @@
                     gv.nodes().click(function () {
                         var $set = $()
                         $set.push(this)
-                        $set = $set.add(gv.linkedFrom(this, true))
-                        $set = $set.add(gv.linkedTo(this, true))
+                        var $direction = $('#streamdirection').val();
+                        if($direction==="bidirectional" || $direction==="downstream") {
```

Comment:
Thanks for the change. It is going to be so handy! I would format the code to keep the style consistent

Reply:
So following conventional js styles:

```javascript
const $direction = $('#streamdirection').val();
if ($direction === 'bidirectional' || $direction === 'downstream') {
    $set = $set.add(gv.linkedFrom(this, true))
}
if ($direction === 'bidirectional' || $direction === 'upstream') {
    $set = $set.add(gv.linkedTo(this, true))
}
```
Just some very minor white space adjustments.

Reply:
Code format changed